src/Lecture1.hs

```diff
@@ -108,7 +121,13 @@ and finds a sum of the numbers inside this string.
 
 The string contains only spaces and/or numbers.
 -}
-strSum str = error "TODO"
+strSum :: String -> Int
+strSum str = let numbers = words str in addNums 0 numbers
+    where
+        addNums :: Int -> [String] -> Int
+        addNums result arrNums
+            | null arrNums = result
+            | otherwise = result + addNums (read (head arrNums)) (tail arrNums) 
```

Comment:
That's a great implementation 👏🏻 

You can parse every string in the list by applying `read` function to each element using the `map` function. After that, you can use the standard function `sum` to find the sum of all elements in the list.

If you use these two functions, you'll be able to implement `strSum` in a single one-line function 🙂